Headlines/src/Controllers/NewsTableViewController.swift

```diff
@@ -555,4 +556,9 @@ class NewsTableViewController: UITableViewController,
         vc.delegate = self
         present(vc, animated: true, completion: nil)
     }
+    
+    // MARK: - TabbedViewController
+    func tabbedViewControllerWasDoubleTapped() {
+        tableView.setContentOffset(CGPoint.zero, animated: true)
+    }
 }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/f539ce350d0d725450191272ea6e28d081f1b565/Headlines/src/Controllers/NewsTableViewController.swift#L564"></span>File should contain 400 lines or less: currently contains 564</td>
    </tr>
  </tbody>
</table>